### ISSUE TITLE
Ruby Syntax Highlighting Crashes VSCode

### ISSUE THREAD

**Issue by JoelBCarter** [Microsoft/vscode#12389]
- VSCode Version: 1.5.2
- OS Version: Windows 10, Version: 1607, OS Build: 14393.0

I first noticed this when working with a `Vagrantfile` which is a Ruby-based DSL, but it appears it any Ruby file will bring my editor to a halt.  `Vagrantfile`s were just nicer because they don't have the `.rb` extension and therefore don't crash until I enable the Ruby syntax highlighting.

Steps to Reproduce:

1. Open Vagrantfile
2. `CTRL` + `K`, `M`
3. Select Ruby from languages
4. Wait for VSCode to freeze

The file doesn't crash if I don't enable Ruby syntax highlighting.  I don't see any errors output to the Dev Console.  I don't (_think_) I have any plugins installed that target Ruby.

**Comment by chrmarti** [comment]
Could not reproduce, could you paste the Vagrantfile you're using?

**Comment by JoelBCarter** [comment]
Here's a simpler way to reproduce for my installation of VSCode:

1. Create a file with the `.rb` suffix
2. Open it in the editor
3. Freeze

**Comment by JoelBCarter** [comment]
I did a little more digging.  It looks like it only occurs when running under my current project directory.  If I create a fresh project directory and add a Ruby file to the new project it's fine.  I can't share my project with you.  I feel this is destined to be closed as not reproducible.  Thanks for your time in advance.

If you can think of any reason why Ruby syntax highlighting under a Node.js/TypeScript project with a few git submodules would crash, I'd love to hear.

**Comment by JoelBCarter** [comment]
Did some more digging and I think I've tracked it down.  I uninstalled the [Chef Extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=Pendrica.Chef) and it's no longer crashing.  I suspect this extension because my team members can correctly open the project in VSCode without any issues and none of them have this extension installed.  I saw some setting in the extension about RuboCop (which I don't think I have installed) so maybe it was trying to call into that and failing for Ruby files?

I'll monitor it for a bit and see if it's a forever fix, but for now I'll close this issue.  Thanks for your time & prompt response.  I'm absolutely loving VSCode!
